backend/tests/integration/test_b07_p4_operational_readiness_e2e.py

```
@@ -0,0 +1,578 @@
+                "hour_start": hour_start,
+                "reason": reason,
+                "disabled_until": disabled_until,
+            },
+        )
+
+
+def _dispatch(task_name: str, payload: LLMTaskPayload) -> dict[str, Any]:
+    async_result = enqueue_llm_task(task_name, payload)
+    result = async_result.get(timeout=120)
+    if not isinstance(result, dict):
+        raise AssertionError(f"Expected dict result for {task_name}, got {type(result).__name__}")
```

Ruff (0.14.14)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)